src/components/ChatList/CommunityChannelItem.tsx

```diff
@@ -13,17 +13,27 @@ import {channelItemStyles as styles} from './ChannelItem.style';
 const CommunityChannelItem = (props: TopicChannelItemProps) => {
   useTopicChannelItemHook(props);
   const {channel: community, onChannelPressed} = props;
-  const channelType = 'COMMUNITY';
   const channelPicture = community?.channelPicture;
   const unreadCount = community?.unreadCount;
   const hasBadge = unreadCount > 0;
 
+  const channelType = () => {
+    switch (community?.channelType) {
+      case 'ANON_TOPIC':
+        return 'ANON_TOPIC';
+      case 'TOPIC':
+        return 'TOPIC';
+      default:
+        return 'TOPIC';
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `channelType` function seems to be redundant. It's returning the same value that it receives as input. You can directly use `community?.channelType` instead of this function.
```diff
-  const channelType = () => {
-    switch (community?.channelType) {
-      case 'ANON_TOPIC':
-        return 'ANON_TOPIC';
-      case 'TOPIC':
-        return 'TOPIC';
-      default:
-        return 'TOPIC';
-    }
-  };
-
-  <ChannelImage.Big type={channelType()} image={channelPicture} />
-  <ChannelImage.Small type={channelType()} />

+  <ChannelImage.Big type={community?.channelType} image={channelPicture} />
+  <ChannelImage.Small type={community?.channelType} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->